docs/how-to/basic_usage.rst

```diff
@@ -0,0 +1,92 @@
+.. _basic_usage:
+
+###################################
+Creating Aldryn News & Blog content
+###################################
+
+Before using Aldryn News & Blog, you need to :ref:`set up your django CMS project appropriately
+<django-cms-setup>` (in short, making sure that you have a page with a NewsBlog apphook - this
+only takes a moment).
+
+This page is where your articles will be listed (see :ref:`multiple_news_sections` for multiple
+lists of articles).
+
+There are various different ways to create a new article:
+
+* From the django CMS toolbar, select **Create** then *New news/blog article*. This opens the
+  django CMS content wizard, providing a quick way to add new content.
+
+* If you're on a page with a News & Blog apphook, select *News & Blog* > *Add new article...* from
+  the django CMS toolbar.
+
+* Add a new article in the Django Admin, in the *Aldryn News and Blog* section.
+
+
+******
+Fields
+******
+
+Most of the fields for an article are self-explanatory, and they behave in logical ways. For
+example, setting the *Publishing date* in the future will publish the item automatically at that
+date/time.
+
+Note that items will not be published until the *Is published* option is explicitly set.
+
+A *Featured* item will be given prominence in lists, such as on the home page of the news section.
+
+
+Meta Options
+============
+
+.. note:
+
+    Fields in the *Meta Options* section should override the default article title, description
+    (taken from the *Lead-in* field) and so on, **but are currently ignored**. This will be fixed
+    in a later revision.
+
+
+Advanced settings
+==================
+
+*Tags* are a set of optional free labels.
```

Comment:
Also, it may be nice to inform developers that they can make tags case-insensitive with the setting: `TAGGIT_CASE_INSENSITIVE = True` and/or provide the link to the Taggit docs for further customization as required.